h5c/vic/src/vic-webapp/src/app/shared/components/vch-general/vch-general.component.ts

```diff
@@ -73,15 +73,17 @@ export class VchGeneralComponent extends VchComponentBase implements OnInit {
 
   protected updateCurrentModel() {
     if (this.form.valid) {
-      this.model.name = this.form.value.name.trim();
-      this.model.debug = this.form.value.debug;
+      const currentModel: VchGeneralView = {
+        name: this.form.value.name.trim(),
+        debug: this.form.value.debug,
+        containerNameConvention: ''
+      };
 
       const prefix = this.form.value.containerNameConventionPrefix.trim();
       const postfix = this.form.value.containerNameConventionPostfix.trim();
 
       if (prefix || postfix) {
-        this.model.containerNameConvention =
-          `${prefix}${this.form.value.containerNameConvention}${postfix}`;
+        currentModel.containerNameConvention = `${prefix}${this.form.value.containerNameConvention}${postfix}`;
       }
 
       // /!* tslint:disable:no-shadowed-variable *!/
```

Comment:
To be honest... I'm not sure.... it was added by Cristian on his original PR where he started working on the reconfigure, but I've almost changed all his work so it can be removed. I did it an I've tested the component and it works as expected.